src/Symfony/Component/Workflow/Dumper/PlantUmlDumper.php

```diff
@@ -160,4 +198,78 @@ private function escape(string $string): string
         // It's not possible to escape property double quote, so let's remove it
         return '"'.str_replace('"', '', $string).'"';
     }
+
+    private function getState(string $place, Definition $definition, Marking $marking = null): string
+    {
+        $placeEscaped = $this->escape($place);
+
+        $stateStyle = $this->getPlaceStyle($place);
+
+        $output = "state $placeEscaped".
+            ($definition->getInitialPlace() === $place ? ' '.self::INITIAL : '').
+            ($marking && $marking->has($place) ? ' '.self::MARKED : '');
+
+        if (isset($stateStyle['background_color'])) {
+            $output .= ' <<'.$this->getColorId($stateStyle['background_color']).'>>';
+        }
+
+        if (isset($stateStyle['description'])) {
+            $output .= ' as '.$place.
+                PHP_EOL.
+                $place.' : '.$stateStyle['description'];
+        }
+
+        return $output;
+    }
+
+    private function getStyledEscapedTransition(string $to, ?array $style): string
+    {
+        $label = $style['label'] ?? $to;
+
+        $color = $style['label_color'] ?? null;
+
+        if (null !== $color) {
+            $to = sprintf(
+                '<font color=%1$s>%2$s</font>',
+                $color,
+                $label
+            );
+        }
+
+        return $this->escape($to);
+    }
+
+    private function getPlaceStyle(string $place): array
+    {
+        return $this->workflowMetadata->getMetadata('dump_style', $place) ?? array();
+    }
+
+    private function getTransitionStyle(Transition $transition): array
+    {
+        return $this->workflowMetadata->getMetadata('dump_style', $transition) ?? array();
+    }
+
+    /**
+     * Color in transition must be prefixed with “#”.
```

Comment:
I think this comment doesn't describe the method but the inner code, so I'd move the comment inside the method. Same for `getColorId()`'s comment.